.github/workflows/build.yml

```diff
@@ -9,25 +9,28 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        tox-env: [py38, py39, py310, linters]
+        tox-env: [py38, py39, py310, docs, linters, package]
         include:
           - tox-env: py38
             python-version: 3.8
           - tox-env: py39
             python-version: 3.9
           - tox-env: py310
             python-version: "3.10"
+          - tox-env: docs
+            python-version: "3.10"
           - tox-env: linters
-            python-version: 3.8
-
+            python-version: "3.10"
+          - tox-env: package
+            python-version: "3.10"
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
Did you consider using dependabot to keep GHA versions up-to-date?

Reply:
Good idea, done. I did not know about depandabot before, thanks for notifying me about this!